lib/ts/recipe/passwordless/recipe.tsx

```diff
@@ -81,15 +83,33 @@ export default class Passwordless extends AuthRecipe<
                 component: (props) => this.getFeatureComponent("linkClickedScreen", props),
             };
         }
+        if (this.config.linkClickedScreen.disableDefaultImplementation !== true) {
+            const normalisedFullPath = this.config.appInfo.websiteBasePath.appendPath(
+                new NormalisedURLPath("/success")
```

Comment:
If we must make this its own page, I don't think `/success` is an appropriate path. Perhaps something like `/close-tab`.